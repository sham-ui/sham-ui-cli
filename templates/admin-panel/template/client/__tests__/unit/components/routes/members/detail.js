import * as directives from 'sham-ui-directives';
import RoutesMembersDetail  from '../../../../../src/components/routes/members/detail.sfc';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( RoutesMembersDetail, {
        directives
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
