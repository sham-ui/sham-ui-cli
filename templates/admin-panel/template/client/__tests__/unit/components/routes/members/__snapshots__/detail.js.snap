// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Constructor": "MembersDetail",
  "Options": Object {
    "ID": "component",
    "directives": Object {
      "EventListener": [Function],
      "disabled": [Function],
      "onclick": [Function],
      "oninput": [Function],
      "onload": [Function],
      "onresize": [Function],
      "onscroll": [Function],
      "onselect": [Function],
      "onsubmit": [Function],
      "ref": [Function],
    },
    "member": Object {},
    "onUpdateSuccess": [Function],
  },
  "html": "
<h6>Member data</h6>
<form class=\\"form-layout\\">
  <!--header-->
  <!--if-->
  <!--ErrorsPanel-->
  <div class=\\"form-group\\">
    <label>Name:</label>
    <input name=\\"name\\" type=\\"text\\" autofocus=\\"\\">
  </div>
  <div class=\\"form-group\\">
    <label>Email:</label>
    <input name=\\"email\\" type=\\"text\\">
  </div>
  <div class=\\"form-group\\">
    <label></label>
    <label class=\\"checkbox\\">
      <input name=\\"is_superuser\\" type=\\"checkbox\\" value=\\"undefined\\"><span>Is Superuser</span></label>
  </div>
  <!--default-->
  <!--default-->
  <button type=\\"submit\\">
    <!--if-->Update
    <!--submit-->
  </button>
  <!--footer-->
</form>
<!--FormContainer-->
<!--if-->
<!--FormWithConfirmModal-->
<!--UpdateDataForm-->
<h6 class=\\"mg-t-30\\">Member password</h6>
<form class=\\"form-layout\\">
  <!--header-->
  <!--if-->
  <!--ErrorsPanel-->
  <div class=\\"form-group\\">
    <label>Password:</label>
    <input name=\\"pass1\\" type=\\"password\\" autofocus=\\"\\">
  </div>
  <div class=\\"form-group\\">
    <label>Confirm password:</label>
    <input name=\\"pass2\\" type=\\"password\\">
  </div>
  <!--default-->
  <!--default-->
  <button type=\\"submit\\">
    <!--if-->Update
    <!--submit-->
  </button>
  <!--footer-->
</form>
<!--FormContainer-->
<!--if-->
<!--FormWithConfirmModal-->
<!--UpdatePasswordForm-->
",
}
`;
